cpp/tests/binning/bin_tests.cu

```diff
@@ -285,16 +293,19 @@ TEST(BinColumnTest, TestBothEdgesWithNullsAfterDifferentAmount)
   fwc_wrapper<float> right_edges{{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}, {1, 1, 1, 1, 1, 1, 1, 1, 1, 0}};
```

Comment:
Can you use typed test instead?

Reply:
Most of the tests you saw were a limited set of tests I used during development. I just rewrote the testing suite in a final pass, and most of the tests are now typed.